src/test_workflow/README.md

```diff
@@ -174,6 +174,17 @@ Internal tools provide dashboards for monitoring cluster behavior during these t
 |Indexing Latency|Consistent during each test iteration|upward trends|
 |Query Latency|Varies based on the query being issued|upward trends|
 
+### Benchmarking Tests
+Runs benchmarking test on a remote opensource OpenSearch cluster. Uses [OpenSearch Benchmark](https://github.com/opensearch-project/OpenSearch-Benchmark) to run benchmark tests.
```

Comment:
test or tests?

you can remove "to run benchmark tests", that's what it does

Reply:
Done!